static/app/views/performance/transactionSummary/transactionEvents/content.tsx

```diff
@@ -180,19 +179,9 @@ const StyledTable = styled('div')`
   flex-grow: 1;
 `;
 
-const StyledSdkUpdatesAlert = styled(GlobalSdkUpdateAlert)`
-  @media (min-width: ${p => p.theme.breakpoints[1]}) {
-    margin-bottom: 0;
-  }
-`;
-
 const SearchRowMenuItem = styled('div')`
   margin-left: ${space(1)};
   flex-grow: 0;
 `;
 
-StyledSdkUpdatesAlert.defaultProps = {
```

Comment:
I removed this and just wrapped the component from whoever renders it - it's less component API to work with and more declarative